docs/notebooks/flax_basics.ipynb

```diff
@@ -0,0 +1,911 @@
+{
```

Comment:
<p>I wouldn't show this case -- people should really treat params as immutable if they're going to take gradients w.r.t. them.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/flax/pull/542/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>Ah no indeed it shouldn't be there, I just tried to see what it would do, but definitely shouldn't be showed here.</p>